subsys/usb/device_next/class/usbd_cdc_acm.c

```diff
@@ -111,6 +111,12 @@ struct cdc_acm_uart_data {
 	struct k_work rx_fifo_work;
 	atomic_t state;
 	struct k_sem notif_sem;
+#ifdef CONFIG_USBD_CDC_ACM_SHORT_PACKET_TERMINATE_ENABLED
+	/* size of packet last tx */
+	uint32_t last_tx_size;
+	/* udc enumerated speed */
+	uint32_t ep_mps_mask;
```

Comment:
Hi @tmon-nordic, ep_mps_mask was wrongly described here, I apologize for the mistake. The ep_mps_mask is saved so that we do not need to query for device speed and calculate the mask every time we want to check whether a transaction requested is actually multiple of EP's Max-Packet-Size. 

I totally agree that we only need a boolean to keep "needs ZLP" information, in case that there isn't more data pushed down to tx_fifo by application layer. will modify that soon and push it up for your review. 

Thanks @tmon-nordic

Reply:
Hi @tmon-nordic, after recent changes, I think this is no longer relevant. Shall we resolve this comment? Thanks 😉